index.html

```diff
@@ -41,19 +40,158 @@ <h1 id="idh1">
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="work s3">
+            <img src="./images/s3.svg" alt="tonic">
+            <h1>Tonic</h1>
+            <div class="info">
+                <h4>CANOPY</h4>
+                <ul class="extra">
+                    <li class="li1"><span>Back End Dev</span></li>
+                    <li class="li2"><span>2015</span></li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="work s4">
+            <img src="./images/s4.svg" alt="tonic">
+            <h1>Multi-Post <br> Stories</h1>
+            <div class="info">
+                <h4>CANOPY</h4>
+                <ul class="extra">
+                    <li class="li1"><span>Back End Dev</span></li>
+                    <li class="li2"><span>2015</span></li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="about-myself">
+            <h1>About <br> Myself</h1>
+            <p id="about-myself">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <div id="socialid">
+                <a href="#" id="connectid">
+                    LET'S CONNECT
```

Comment:
- [ ] kindly use a header tag `hx`  for the` lets connect` as its a sub heading as well

Reply:
@tufoinnkuo10 Please could you give me an example or more detail about the hx tag? or it has to be an specific number from h1-h6 tag?

Reply:
@CarlosIgreda an h2/h3 tag will do...

Reply:
Done @tufoinnkuo10 . I've already changed it to an h3 tag.